server/api/caseNotesApi.js

```diff
@@ -19,6 +40,21 @@ const caseNotesApiFactory = client => {
       .then(processResponse(context))
       .catch(processError)
 
+  const getCaseNotes = (context, offenderNo, { perPage, pageNumber, type, subType, startDate, endDate }) => {
+    const query = {
```

Comment:
think the logic in this class needs testing, can't see any tests for extracting the fields and converting the dates etc - especially end of day...